charts/osm/values.yaml

```diff
@@ -18,6 +18,7 @@ OpenServiceMesh:
   imagePullSecrets: []
   # -- Envoy sidecar image
   sidecarImage: envoyproxy/envoy-alpine:v1.19.0
+  sidecarWindowsImage: envoyproxy/envoy-windows:v1.19.0
```

Comment:
nit: add comment similar to L19 per property here for clarity

Reply:
can we get this one merged and I can do this in a follow up today